pkg/objstore/inmem/inmem.go

```diff
@@ -123,6 +122,10 @@ func (b *Bucket) GetRange(_ context.Context, name string, off, length int64) (io
 		return ioutil.NopCloser(bytes.NewReader(file[off:])), nil
 	}
 
+	if length <= 0 {
```

Comment:
I think the error message does not agree with the condition here. Message says “smaller than 0” but the condition is <=0

Reply:
Done